src/App.tsx

```diff
@@ -1,12 +1,71 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
+import classNames from 'classnames';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { GoodsList } from './Components/GoodsList';
 
-const App: React.FC = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
+  const [isActiveAll, setIsActiveAll] = useState(false);
+  const [isActiveFive, setIsActiveFive] = useState(false);
+  const [isActiveRed, setIsActiveRed] = useState(false);
+
+  async function goodsAllHandler() {
+    setGoods(await getAll());
+    setIsActiveAll(true);
+    setIsActiveFive(false);
+    setIsActiveRed(false);
+  }
+
+  async function goodsFiveHandler() {
+    setGoods(await get5First());
+    setIsActiveAll(false);
+    setIsActiveFive(true);
+    setIsActiveRed(false);
+  }
+
+  async function goodsRedHandler() {
+    setGoods(await getRedGoods());
+    setIsActiveAll(false);
+    setIsActiveFive(false);
+    setIsActiveRed(true);
+  }
+
+  return (
+    <div className="content">
+      <div className="buttons has-addons">
+        <button
+          type="button"
+          onClick={goodsAllHandler}
+          className={classNames('button', {
+            'is-warning': isActiveAll,
```

Comment:
If you need it for classnames, will be better to create one variable that will be store string like (activeAll, activeFive, ...), and use this variable here